docs/data/destinations/appsflyer.md

```diff
@@ -18,11 +18,9 @@ Amplitude Data's AppsFlyer integration lets you stream your Amplitude event data
 
 Keep these things in mind when sending events to AppsFlyer:
 
-- A user must exist in AppsFlyer to send events for them. Amplitude sends the event data with the `user_id`, which must match the `customer_user_id` within AppsFlyer. Make sure that the AppsFlyer `customer_user_id` and the Amplitude `user_id` match.
-- Amplitude forwards all event, user, and group properties as key value pairs within `eventValue`.
-- Amplitude sends the `os` property. AppsFlyer recommends including it because for iOS apps, if the `os` property isn't included for iOS apps, AppFlyer records the data as coming from a device running iOS 14.5.
-- Amplitude sends device identifiers if available. See the [AppFlyer documentation](https://support.appsflyer.com/hc/en-us/articles/207034486#payload-parameters) for more information about what happens if you don't send a device identifier.
-- Amplitude attempts to send the device identifiers listed in the [AppFlyer documentation](https://support.appsflyer.com/hc/en-us/articles/207034486#payload-parameters).
+- A user must exist in AppsFlyer to send events for them. Appsflyer requires the `appsflyer_id` corresponding to each user to be sent. Use the `Mappings` tab in the sync setup to specify which Amplitude property to map to `appsflyer_id`. Specify a mapping for the `customer_user_id` AppsFlyer property for best accuracy.
+- Amplitude forwards all properties selected to be sent as key value pairs within `eventValue`.
+- Appsflyer recommends sending the `OS` field, as well as other device identifiers such as various ad identifiers and device ID. See the [AppFlyer documentation](https://support.appsflyer.com/hc/en-us/articles/207034486#payload-parameters) for their recommendations on which properties to send. Use the `Mappings` tab in the sync setup to choose which properties to forward.
```

Comment:
I agree with the idea of keeping this section general, but I also think it's important for them to know this information so that they don't skip the mappings tab and mappings instructions. Maybe could switch them out on some of the other destinations but for ex. Appsflyer does not require OS but they **highly** recommend it lol so want to make sure the user knows that.